Makefile

```diff
@@ -134,6 +134,22 @@ generate-bundle: regenerate $(OPM)
 	$(OPERATOR_SDK) bundle validate --verbose bundle
 .PHONY: generate-bundle
 
+# Generate bundle manifests and metadata, then validate generated files.
+VERSION ?= 4.7.0
+# Options for 'bundle-build'
+BUNDLE_CHANNELS := --channels=4.7
+BUNDLE_DEFAULT_CHANNEL := --default-channel=4.7
```

Comment:
I suggest to re-use `OCP_VERSION` here. It is already a widespread hassle to catch all OCP versions on each release.

Reply:
Updated makefile to use `OCP_VERSION`